### ISSUE TITLE
Syndicated Hacks blog posts should have original card image

### ISSUE THREAD

**Issue by valgrimm** [mdn/developer-portal#1175]
Hacks Blog post cards are not autopopulated with featured image.
However, Mixed Reality post cards are autopopulated with featured image.
AC is that Hacks Blog cards are autopopulated with post featured image. 

Hacks is Wordpress 5.3.2 running Hax theme
Have to ask Josh Marinacci what Mixed Reality Blog is . . . if that would be useful.

Not sure what the syndication code hooks into, so to save digging around, need to ask Steve for that info.

**Comment by valgrimm** [comment]
This is also true for Add Ons blog, but less important than Hacks at moment. 
Example
https://blog.mozilla.org/addons/2020/02/06/ublock-origin-for-firefox-android-nightly/

**Comment by stevejalim** [comment]
Heya @valgrimm 

Have had a little look into this and my first thought is "Oh, Mixed Reality is providing preview/thumbnail images in its RSS feed, but Hacks and Addons are not"

**An example `item` from the Mixed Reality feed** at https://blog.mozvr.com/rss/
<img width="890" alt="Screenshot 2020-02-07 at 11 07 38" src="https://user-images.githubusercontent.com/101457/74024858-944bac80-499a-11ea-8896-2870d9f65e74.png">

**An example `item` from the Hacks feed** at https://hacks.mozilla.org/feed/

<img width="788" alt="Screenshot 2020-02-07 at 11 07 50" src="https://user-images.githubusercontent.com/101457/74024932-c0672d80-499a-11ea-8440-a9f6c8627802.png">


**An example `item` from the Add Ons feed** at https://blog.mozilla.org/addons/feed/

<img width="893" alt="Screenshot 2020-02-07 at 11 07 21" src="https://user-images.githubusercontent.com/101457/74025033-fc9a8e00-499a-11ea-9af4-cc6572c8b496.png">

Note that the RSS feed consumer will only look for images in specific nodes (eg `media`) and it doesn't scrape images from any preview of the main content.

So, unless there's a way to get the Hacks and Addons blogs to also include media in their RSS feeds, it might have to just be a manual image addition at the point of article moderation...

**Comment by valgrimm** [comment]
Yeah, I did View Source on them and was tired and unaware of the standard for this so then just said "bleh, Steve will find this faster than me"
I'm fine with the manual workaround, but perhaps that is something to fix on the Hacks and Mozilla Blogs side. Although on Hacks tech support is basically the goodwill of someone taking extra time to help . . . so I'll leave this lie and go manual. Closing ticket.
